format/spec.md

```diff
@@ -779,7 +779,9 @@ When two commits happen at the same time and are based on the same version, only
 
 #### File System Tables
 
-An atomic swap can be implemented using atomic rename in file systems that support it, like HDFS or most local file systems [1].
+_Note: This file system based scheme to commit a metadata file is **deprecated** and will be removed in a future version of this spec. The scheme is **unsafe** in object stores and local file systems._
```

Comment:
For deprecations we should always indicate a target for removal (e.g. V4 of the spec since that would be two releases away).  We can always push the target back but leaving it unclear means that users and even maintainers don't have clarity.  It's also best practice to include the "appropriate alternative" in the deprecation message so it isn't left to the reader to find what the recommended path is.

Reply:
Updated.